docs/articles/documentation/reference/configuration-file.md

```diff
@@ -751,11 +745,106 @@ Injects scripts into pages visited during the tests. Use this property to introd
 }
 ```
 
-> Relative paths are resolved against the current working directory.
+### Inject a JavaScript File
```

Comment:
To editors: no text was added or modified in this file except for the headers. Other changes are technical.